### ISSUE TITLE
Listen to stations under "Your Servers"

### ISSUE THREAD

**Issue by KatyLB** [segler-alex/RadioDroid#851]
I _assume_ this is possible, but I'm making a report because I: A) Can't figure out how despite looking through the app, and B) Feel it would be beneficial to make it easier to figure out.

RadioDroid has the ability to add a station under "Your Servers". No play button is given and I can't tell how you can listen to these servers in the app.

**Comment by werman** [comment]
This is for MPD (music player daemon) servers, e.g. you could launch MPD server on your PC and play station there from RadioDroid.

It's not possible to add custom servers in the RadioDroid at the moment (you could add station to the public database http://www.radio-browser.info/gui/#!/ which RadioDroid uses).

**Comment by KatyLB** [comment]
Oh, ok. I was trying to use the app to listen to a test stream so I didn't want to add it to the public database.

In that case, could this be a feature request?

**Comment by werman** [comment]
Yes, it gets asked from time to time and it indeed would be useful, just no one got around to make it.

**Comment by hamedsbt** [comment]
How add station's stream-address manually?!

**Comment by Vingolf** [comment]
> How add station's stream-address manually?!

As werman allready pointed out, you can add a station on 
http://www.radio-browser.info/gui/#!/add and then use your new created station on RadioDroid.

**Comment by Wikinaut** [comment]
I think that the idea - which I support - is to add a station locally in the app.

**Comment by werman** [comment]
Yes, the idea is to add a station locally. I think I did half of this work locally, but had no time to finish, may be I'll do it soon...
